api/Gemfile

```diff
@@ -4,10 +4,12 @@ source "https://rubygems.org"
 
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
-gem 'roda'
+gem 'bcrypt'
 gem 'dotenv'
+gem 'json'
+gem 'roda'
+gem 'rubocop', require: false
 gem 'sequel'
-gem 'bcrypt'
 gem 'sqlite'
-gem 'rubocop', require: false
-gem 'json'
+gem 'sqlite3'
```

Comment:
Do we need both?

Reply:
That is a good question. Please investigate. I think `sequel` will run into issues without `sqlite`.
Make sure that you don't have them installed globally when testing. I think following command will show you if they are installed globally:

```bash
$ gem list | grep sqlite
```